pureskillgg_dsdk/tome/manifest.py

```diff
@@ -34,17 +42,30 @@ def end_page(self, page_number):
             "number": page_number,
             "keyset": {
                 "keyset": content_name(page_number, "keyset"),
-                "key": "/".join([self._tome_name, content_name(page_number, "keyset")]),
+                "key": "/".join(
+                    [
+                        self._ds_type,
```

Comment:
I think the key needs to start with `tome` just list the csds key starts with `csds`.

Reply:
👍